source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,32 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the `updateOne() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne>`_ method. 
+``updateOne()`` takes a filter for a document, and updates the first document
+that matches the filter.
+
+Create an Options object to specify additional options. Specify the
+``upsert`` option to create a new document if no documents match the
+filter.  To receive acknowledgment in the response from MongoDB that the
+write operation was performed on the specified number of :manual:`mongod
+</reference/program/mongod>` servers, set the ``w`` option. 
```

Comment:
Reversing this line might make it easier to track -- I know the first half of the sentence was a bit confusing for me to parse.

Reply:
Note: I'm making a new PR for this, but to address your comment, I'm removing the 'w' opt altogether, we don't want to talk about write concern in these docs.